recipe/meta.yaml

```diff
@@ -6,26 +6,27 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/argcomplete-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/argcomplete-{{ version }}.tar.gz
   sha256: eb1ee355aa2557bd3d0145de7b06b2a45b0ce461e1e7813f5d066039ab4177b4
 
 build:
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   noarch: python
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ python_min }}
     - setuptools >=67.7.2
     - setuptools-scm >=6.2
     - pip
   run:
-    - python >=3.8
+    - python >={{ python_min }}
```

Comment:
Previously this had a Python min of 3.8, which aligns with [the upstream `3.5.1` Python min]( https://github.com/kislyuk/argcomplete/blob/v3.5.1/pyproject.toml#L9 )

If we would like to keep this minimum, would suggest either adding the following above

```yaml
{% set python_min = "3.8" %}
```

...or add a new file `recipe/conda_build_config.yaml` with the following (and [re-render]( https://conda-forge.org/docs/maintainer/updating_pkgs/#rerendering-feedstocks ))

```yaml
python_min:
- '3.8'
```